lib/services/stats/statsRegistry.js

```diff
@@ -72,6 +72,77 @@ function globalLoad(values, callback) {
     callback(null);
 }
 
+/**
+ * Chooses the appropiate content type and version based on Accept header
+ * 
+ * @param {String} accepts The accepts header
+ */
+function matchContentType(accepts) {
+    const requestedType = [];
+    for (const expression of accepts.split(',')) {
+        const parts = expression.split(';').map((part) => part.trim());
+        const mediaType = parts[0];
+        let version = null;
+        let charset = null;
+        let preference = null;
+        for (let part of parts.slice(1)) {
+            if (part.startsWith('version=')) {
+                version = part.substring(8).trim();
+            } else if (part.startsWith('charset=')) {
+                charset = part.substring(8).trim();
+            } else if (part.startsWith('q=')) {
+                preference = parseFloat(part.substring(2).trim());
```

Comment:
In order to avoid magin numbers maybe it's better this way:

```suggestion
        let versionTokenLength = 'version=.length;
        let qTokenLength = 'q='.length;
        for (let part of parts.slice(1)) {
            if (part.startsWith('version=')) {
                version = part.substring(versionTokenLength ).trim();
            } else if (part.startsWith('charset=')) {
                charset = part.substring(versionTokenLength ).trim();
            } else if (part.startsWith('q=')) {
                preference = parseFloat(part.substring(qTokenLength).trim());
```

Reply:
fix in 679813e4f9e0d59dd175805f52a5c43ba06ef220